jupyterbook/content/Code Gallery/data_access_notebooks/2016-10-12-fetching_data.ipynb

```diff
@@ -867,9 +880,9 @@
    "name": "python",
    "nbconvert_exporter": "python",
    "pygments_lexer": "ipython3",
-   "version": "3.7.1"
+   "version": "3.9.1"
   }
  },
  "nbformat": 4,
- "nbformat_minor": 1
+ "nbformat_minor": 4
 }
```

Comment:
@ocefpaf, in this particular notebook, the difference is because this is the original notebook that is unrun -used in the Demo Data Center- and older PR had a re-run notebook.